test/spec/invertedIndexSpec.js

```diff
@@ -32,61 +26,55 @@ describe('InvertedIndex', () => {
       expect(InvertedIndex.validateFile(notArray)).toBeFalsy();
       expect(InvertedIndex.validateFile(wrongData)).toBeFalsy();
     });
-    it('should return the file content if a valid file was uploaded', () => {
-      expect(InvertedIndex.validateFile(validFile)).not.toBeFalsy();
-      expect(InvertedIndex.validateFile(simpleValidFile)).not.toBeFalsy();
+
+    it('should not return false if a valid file was uploaded', () => {
+      expect(InvertedIndex.validateFile(books)).not.toBeFalsy();
+      expect(InvertedIndex.validateFile(shortStories)).not.toBeFalsy();
+      expect(typeof InvertedIndex.validateFile(books)).toEqual('object');
+    });
+
+    it('should return the file content if the file is valid', () => {
+      expect(InvertedIndex.validateFile(books)).toEqual(books);
+      expect(InvertedIndex.validateFile(shortStories))
+      .toEqual(shortStories);
     });
   });
 
   describe('Tokenize', () => {
-    it('return an array of unique words', () => {
+    it('should return an array of unique words', () => {
       expect(InvertedIndex.tokenize('tony is humble tony'))
       .toEqual(['tony', 'is', 'humble']);
     });
+
     it('should remove invalid characters', () => {
       expect(InvertedIndex.tokenize('$A%^n*del)a %i=s @a$w&es$om#e%'))
       .toEqual(['andela', 'is', 'awesome']);
     });
-    it('should return case-insensitive result', () => {
-      expect(InvertedIndex.tokenize('HI')).toEqual(['hi']);
+
+    it('should return lowercase words', () => {
+      expect(InvertedIndex.tokenize('HEllO WORLD')).toEqual(['hello', 'world']);
     });
   });
 
   describe('CreateIndex', () => {
-    beforeAll(() => {
-      this.filename = 'simpleValidFile.json';
-      this.check = index.createIndex(simpleValidFile, this.filename);
+    it('creates an index object from the file content', () => {
+      expect(index.createIndex(shortStories, filename)).toBeTruthy();
+      expect(typeof index[filename]).toEqual('object');
```

Comment:
I think this can be moved to another test case checking that the type of index created is an object.